protractor/e2e/integration/second-page.spec.ts

```diff
@@ -61,13 +61,28 @@ describe('tests for second page', async () => {
             expect(await secondPage.infoSubmitButton.isDisplayed()).toBeTruthy();
             await secondPage.infoSubmitButton.click();
 
+    });
 
 
+    fit('check the blog (homework)', async () => {
 
+        //await browser.get('https://ng-book.com/2/');
 
-    });
-
-
+        await secondPage.menuItemBlog.click();
+        expect(await browser.getCurrentUrl()).toContain('https://blog.ng-book.com/');
+        expect(await secondPage.header.count()).toEqual(20);
+        expect(await secondPage.description.count()).toEqual(20);
+        expect(await secondPage.link.count()).toEqual(20);
+        expect(await secondPage.linkFirstPage.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkOne.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkTwo.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkNext.isDisplayed()).toBeTruthy();
+        expect(await secondPage.linkLastPage.isDisplayed()).toBeTruthy();
+        await secondPage.linkTwo.click();
+        expect(await secondPage.getCurrentUrl()).toContain('/page/2/');
+        expect(await secondPage.blockPost.isDisplayed()).toBeTruthy();
+        expect(await secondPage.blockPost.get(0).getText());
```

Comment:
You check nothing in this line. You just getting text and don't mind what it will be. Even if it'll be undefined. SO, after your expect() always should be some expected value, like .toBe(), .toContain(), .toEqual(), etc.